appveyor.yml

```diff
@@ -31,4 +31,4 @@ build_script:
 
 test_script:
   - SET SPEC_OPTS=--format progress
-  - bundle exec rake spec
+  - bundle exec rake spec:appveyor
```

Comment:
Do we need a special task? How about `bundle exec rake spec:unit spec:functional`? That should run the unit tests and if they pass run the functional tests.